### DISCUSSION TITLE
갤럭시 아이폰 쉐이더 문제

### DISCUSSION THREAD

**Discussion by LeeYongSu1** [naverz/zepeto-studio-kor#358] [5. Q&A]
안녕하세요

 유니티로 구축한 월드의 커스텀 쉐이더가 아이폰과 안드로이드의 화면에서 서로 다르게 나오는 현상을 겪고 있습니다.

물 커스텀 쉐이더에 큐브맵 스카이박스가 비추도록 구현을 하였으나 안드로이드에서는 잘 동작을 하나 아이폰에서 동작을 하지 않는 

모습입니다.

**유니티 화면**
![화면 캡처 2022-02-10 141642](https://user-images.githubusercontent.com/80555875/153342324-679c060f-f401-4487-a0c9-b8634f1bf0a0.png)

**안드로이드**
![KakaoTalk_20220209_160130274_01](https://user-images.githubusercontent.com/80555875/153342501-cb40b3f4-e740-4c73-91f4-ba0af9410200.jpg)

**아이폰**
![KakaoTalk_20220209_160130274](https://user-images.githubusercontent.com/80555875/153342719-ed4e59a5-abf7-496e-9faa-05e9fa8a8e7e.jpg)

어떻게 해결을 해야 좋을지 문의 드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
Edit> Project Setting > Quality에서 PC와 iOS, Android 퀄리티 차이가 다른게 기본이라서
퀄리티 차이가 나는 이유도 있습니다.

⬇️ 기본적으로 퀄리티가 다르게 세팅되어 있습니다. 

<img width="631" alt="스크린샷 2022-03-23 오후 6 28 29" src="https://user-images.githubusercontent.com/100118650/159667427-bec7b319-fdaa-4bbd-8a11-ab934f75e2a1.png">

⬇️ 이렇게 동일하게 맞춰주세요!

<img width="631" alt="스크린샷 2022-03-23 오후 6 28 47" src="https://user-images.githubusercontent.com/100118650/159669002-6adb6da3-3e5d-48f8-a24c-077852ff747e.png">
